CMakeLists.txt

```diff
@@ -30,6 +30,19 @@ FetchContent_Declare(
   GIT_TAG        f9ff166845a59327eda431af82ee85a9c7532c5d
 )
 
+if(NOT WIN32)
+  if(EXISTS ${CMAKE_SYSROOT}/usr/lib/x86_64-linux-gnu/libvulkan.so)

```

Comment:
Is there a reason to not try find_package(Vulkan) first and only do this if !Vulkan_FOUND or does find_package find the wrong thing? I know I had some weird issues on GH actions runners with CUDA.